packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -151,6 +157,12 @@ describe('<DataGrid /> - Keyboard', () => {
     expect(getActiveCell()).to.equal('1-0');
     fireEvent.keyDown(document.activeElement!, { key: 'ArrowUp' });
     expect(getActiveCell()).to.equal('0-0');
+    fireEvent.keyDown(document.activeElement!, { key: 'ArrowUp' });
+    expect(getActiveCell()).to.equal('null-1');
```

Comment:
Add `getActiveHeaderCell` to check focus on header cells?

Reply:
The thing is that cells do not have `rowIndex`, that's why there is a `null`. Adding one would only benefit the test as it isn't used anywhere else in the logic.

Regarding, the `getActiveHeaderCell` -> it can be a possible solution and it will only return the `colIndex` so basically,
`getActiveHeaderCell() === 'n' \\ where n is the value of the colIndex`

Reply:
Updated